utils/installer/install.sh

```diff
@@ -138,8 +177,16 @@ function print_missing_dep_msg() {
 }
 
 function check_dep() {
-  if ! command -v "$1" &>/dev/null; then
-    print_missing_dep_msg "$1"
```

Comment:
I get the idea, but it has a few problems.
- it's used once, and pretty unlikely to be needed in the future
- the function name doesn't even reflect the new behaviour
- it created an implicit global variable `$check_dep_return` but you're ignoring the return value, maybe it was the one actually intended to be used.

Overall I actually think it's better to retain the original function since this is not gonna save us anything. It will be more clear if you just check each of them individually.

Reply:
Sure, I can make the change, maybe it would be nice to have consensus on the order of dependencies -- `npm` as first choice or `yarn`. That way it could be one commit.

Reply:
IMO the whole function serves no purpose, check_dep is already achieved by `command -v`.
You would just need to move the loop in `install_nodejs_deps` and print a message if it returns a non 0 code
Also, the function should not exit

Reply:
@Tastyep Ok. I can change it to that.

I think the current behavior is to fail (exit) if npm is not installed and the user answer was yes. Should I retain that?

Reply:
https://github.com/LunarVim/LunarVim/pull/1565/files#r711616459